packages/browser-wallet/src/popup/shared/IdCard/IdCard.stories.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/function-component-definition, import/no-extraneous-dependencies */
+import React, { useState } from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import IdCard from './IdCard';
+
+export default {
+    title: 'Shared/IdCard',
+    component: IdCard,
+} as ComponentMeta<typeof IdCard>;
+
+const Template: ComponentStory<typeof IdCard> = (args) => {
+    // eslint-disable-next-line react/destructuring-assignment
+    const [name, setName] = useState(args.name);
+    return <IdCard {...args} name={name} onNameChange={setName} />;
+};
+
+export const Pending = Template.bind({});
+Pending.args = {
+    name: 'Name',
+    status: 'pending',
+    provider: <>DTS</>,
```

Comment:
There should be an example with an image as the provider as that is what the Figma sketches use.